libs/core/include/core/serializers/exception.hpp

```diff
@@ -54,6 +57,10 @@ class SerializableException : public std::exception
 
   void StackTrace() const;
```

Comment:
If the ticket does not already exist, let's make one to deprecate StackTrace?

Reply:
👍